modules/exploits/windows/http/desktopcentral_deserialization.rb

```diff
@@ -14,60 +14,75 @@ class MetasploitModule < Msf::Exploit::Remote
   include Msf::Exploit::FileDropper
 
   def initialize(info = {})
-    super(update_info(info,
-      'Name'            => 'ManageEngine Desktop Central Java Deserialization',
-      'Description'     => %q{
-        This module exploits a Java deserialization vulnerability in the
-        getChartImage() method from the FileStorage class within ManageEngine
-        Desktop Central versions < 10.0.474. Tested against 10.0.465 x64.
-
-        "The short-term fix for the arbitrary file upload vulnerability was
-        released in build 10.0.474 on January 20, 2020. In continuation of that,
-        the complete fix for the remote code execution vulnerability is now
-        available in build 10.0.479."
-      },
-      'Author'          => [
-        'mr_me', # Discovery and exploit
-        'wvu'    # Module
-      ],
-      'References'      => [
-        ['CVE', '2020-10189'],
-        ['URL', 'https://srcincite.io/advisories/src-2020-0011/'],
-        ['URL', 'https://srcincite.io/pocs/src-2020-0011.py.txt'],
-        ['URL', 'https://twitter.com/steventseeley/status/1235635108498948096'],
-        ['URL', 'https://www.manageengine.com/products/desktop-central/remote-code-execution-vulnerability.html']
-      ],
-      'DisclosureDate'  => '2020-03-05', # 0day release
-      'License'         => MSF_LICENSE,
-      'Platform'        => 'win',
-      'Arch'            => [ARCH_CMD, ARCH_X86, ARCH_X64],
-      'Privileged'      => true,
-      'Targets'         => [
-        ['Windows Command',
-          'Arch'        => ARCH_CMD,
-          'Type'        => :win_cmd
+    super(
+      update_info(
+        info,
+        'Name' => 'ManageEngine Desktop Central Java Deserialization',
+        'Description' => %q{
+          This module exploits a Java deserialization vulnerability in the
+          getChartImage() method from the FileStorage class within ManageEngine
+          Desktop Central versions < 10.0.474. Tested against 10.0.465 x64.
+
+          "The short-term fix for the arbitrary file upload vulnerability was
+          released in build 10.0.474 on January 20, 2020. In continuation of
+          that, the complete fix for the remote code execution vulnerability is
```

Comment:
See comment above, same terminology fixes should be applied here.

Reply:
See responses above.

Reply:
Yep can see this was a quote now, so all good.